cli/list.go

```diff
@@ -331,7 +331,8 @@ func getContainers(ctx context.Context, context *cli.Context) ([]fullContainerSt
 
 			uid, err := getDirOwner(container.RootFs)
 			if err != nil {
-				return nil, err
+				fmt.Printf("failed to get container %s rootfs: %s\n", ociState.ID, err)
```

Comment:
Could you change this to something like:

```go
fmt.Fprintf(os.Stderr, "WARNING: failed to get container %s rootfs: %s\n", ociState.ID, err)
```

Reply:
Updated.